jib-maven-plugin/src/test/resources/projects/simple/pom.xml

```diff
@@ -52,6 +52,9 @@
               <key1>value1</key1>
               <key2>value2</key2>
             </labels>
+            <volumes>
+              <volume>/var/log</volume>
```

Comment:
I would recommend adding another one just to sanity-check the list works as intended.